<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Eleazar Meza</author>
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Romano</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pixmaps/pixmaps.qrc">
    <normaloff>:/icons/romano-icon.png</normaloff>:/icons/romano-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Camiones en tránsito</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Actualizar lista</string>
        </property>
        <property name="icon">
         <iconset resource="pixmaps/pixmaps.qrc">
          <normaloff>:/icons/refresh-tickets.png</normaloff>:/icons/refresh-tickets.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="ticketsTableView">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewReception"/>
   <addaction name="separator"/>
   <addaction name="actionNewDispatch"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="sessionToolbar">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLogout"/>
   <addaction name="actionUserName"/>
  </widget>
  <action name="actionNewReception">
   <property name="icon">
    <iconset resource="pixmaps/pixmaps.qrc">
     <normaloff>:/icons/reception-icon.png</normaloff>:/icons/reception-icon.png</iconset>
   </property>
   <property name="text">
    <string>Nueva recepción</string>
   </property>
   <property name="toolTip">
    <string>Crear nuevo ticket de recepción</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNewDispatch">
   <property name="icon">
    <iconset resource="pixmaps/pixmaps.qrc">
     <normaloff>:/icons/dispatch-icon.png</normaloff>:/icons/dispatch-icon.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo despacho</string>
   </property>
   <property name="toolTip">
    <string>Crear nuevo ticket de despacho</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="pixmaps/pixmaps.qrc">
     <normaloff>:/icons/log-out.png</normaloff>:/icons/log-out.png</iconset>
   </property>
   <property name="text">
    <string>Cerrar sesión</string>
   </property>
   <property name="toolTip">
    <string>Cerrar sesión</string>
   </property>
  </action>
  <action name="actionUserName">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pixmaps/pixmaps.qrc"/>
 </resources>
 <connections/>
</ui>
